### DISCUSSION TITLE
英语支援

### DISCUSSION THREAD

**Discussion by ivan-ef** [diylxy/LiThermal#6] [General]
這個項目只有中文嗎？或者說英語的使用者可以使用它嗎？

**Comment by lxydiy** [comment]
因为海康tb4117模块只有国内有，如果只是个人diy需求的话没必要做i18n

**Reply by ivan-ef** [reply]
> 因为海康tb4117模块只有国内有，如果只是个人diy需求的话没必要做i18n

我在中國購買並在國外使用。

**Comment by ivan-ef** [comment]
> 因为海康tb4117模块只有国内有，如果只是个人diy需求的话没必要做i18n

您是否同意在商业基础上将固件翻译成另一种语言？让我知道你的想法。微信 ivanefimov

**Comment by lxydiy** [comment]
允许在声明原作者且遵守GPL3.0协议的前提下商用，但因此带来的一切问题请自行解决
